.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+*.log
```

Comment:
add empty line to the end of file

Reply:
hmm, it looks like this file already has empty end line, but you can't see it here)